cli/azd/internal/appdetect/spring_boot.go

```diff
@@ -229,6 +244,38 @@ func detectEventHubsAccordingToSpringCloudEventhubsStarterDependency(
 	}
 }
 
+func detectEventHubsAccordingToSpringIntegrationEventhubsMavenDependency(
+	azdProject *Project, springBootProject *SpringBootProject) {
+	var targetGroupId = "com.azure.spring"
+	var targetArtifactId = "spring-cloud-azure-starter-integration-eventhubs"
+	if hasDependency(springBootProject, targetGroupId, targetArtifactId) {
+		newDep := AzureDepEventHubs{
+			// eventhubs name is empty here because no configured property
```

Comment:
Will it return error when run `azd provision` when provision event hub without event hub name?

Reply:
No, because this dependency that has no event hubs will be prompted to input the event-hub-name later, so it will not spread to the `azd provision` stage.